backend/src/main/java/develup/api/DiscussionApi.java

```diff
@@ -49,4 +49,12 @@ public ResponseEntity<ApiResponse<DiscussionResponse>> createDiscussion(
 
         return ResponseEntity.ok(new ApiResponse<>(response));
     }
+
+    @GetMapping("/discussions/mine")
+    @Operation(summary = "나의 디스커션 목록 조회 API", description = "내가 제출한 디스커션 목록을 조회합니다.")
```

Comment:
[Request Changes]

아래와 같은 느낌은 어떨까요?

```suggestion
    @Operation(summary = "나의 디스커션 목록 조회 API", description = "내가 작성한 디스커션 목록을 조회합니다.")
```